app/assets/javascripts/hubstats/application.js

```diff
@@ -14,45 +14,83 @@
 //= require jquery_ujs
 //= require select2
 //= require hubstats/bootstrap
+//= require bootstrap-datepicker
 //= require_tree .
 
 $(document).ready( function() {
+  defaultDateRange()
   setDateRange()
 });
 
+function defaultDateRange() {
+  $('.input-daterange').datepicker({
+    "todayHighlight": true,
+    "endDate": "Today",
+    "todayBtn": true
+  });
+};
+
 function setDateRange() {
-  var index = readCookie("hubstats_index") || 2;
-  var timer = document.getElementById("time-select");
+  if ((readCookie("hubstats_index") === "null~~null") || (readCookie("hubstats_index") === null)) {
+    var index = getDefaultDateRange();
```

Comment:
It is a Javascript best practice to define all variables at the top of your function because variables always have function scope.

Also, this variable no longer stores an index.  Consider renaming it.